packages/cactus-plugin-ledger-connector-besu/README.md

```diff
@@ -1,9 +1,103 @@
-# `@hyperledger/cactus-plugin-ledger-connector`
+# `@hyperledger/cactus-plugin-ledger-connector-besu`
 
-> TODO: description
+This plugin provides `Cactus` a way to interact with Besu networks. Using this we can perform:
+* Deploy Smart-contracts through bytecode.
+* Build and sign transactions using different keystores.
+* Invoke smart-contract functions that we have deployed on the network.
+## Summary
+
+  - [Getting Started](#getting-started)
+  - [Usage](#usage)
+  - [Runing the tests](#running-the-tests)
+  - [Built With](#built-with)
+  - [Contributing](#contributing)
+  - [License](#license)
+  - [Acknowledgments](#acknowledgments)
+
+## Getting Started
+
+Clone the git repository on your local machine. Follow these instructions that will get you a copy of the project up and running on
+your local machine for development and testing purposes.
+
+### Prerequisites
+
+In the root of the project to install the dependencies execute the command:
+```
+npm run comfigure
+```
+
+### Compiling
+
+In the plugin root folder (/packages/cactus-plugin-ledger-connector-besu), run this command to compile the plugin and create the dist directory:
+> Check this
+```
+npm run tsc
```

Comment:
@AzaharaC Does this really work in the plugin root folder instead of the project root?

Reply:
I have been checking this and it is true that if it is run from the plugin root, not work.

Reply:
@AzaharaC Then I'd recommend rephrasing it to make sure we don't get bug reports about it. If the readme says that it has to be run from the project root (not the package root) then it becomes correct IMO.